src/client.ts

```diff
@@ -2044,6 +2089,16 @@ export class StreamChat<
    */
   _isUsingServerAuth = () => !!this.secret;
 
+  _isRunningInNode = () => {
```

Comment:
this is not something that changes in runtime, better to initialize it in the constructor

Reply:
actually, you can reuse `this.node` value